ui/views_test.py

```diff
@@ -252,6 +252,29 @@ def test_404_error_context_logged_out(self):
             assert response.context['authenticated'] is False
             assert response.context['name'] == ""
 
+    def test_500_error_context_logged_in(self):
```

Comment:
after trying few approaches, I realized that it is not currently possible. 
See https://code.djangoproject.com/ticket/18707
and it looks they are non going to implement it any time soon